docs/runbooks/SSPCommonTemplatesModificationReverted.md

```diff
@@ -0,0 +1,21 @@
+# SSPCommonTemplatesModificationReverted
+
+## Meaning
+
+This alert indicates that one or more common templates were changed by the ssp operator as part of its reconciliation procedure.
+If the operator was upgraded this alert just indicates the common templates were updated to the latest version.
+
+## Impact
+
+If any changes were made to a common template manually or by a script they were reverted by the operator.
+
+## Diagnosis
+
+If the operator was upgraded this alert is to be expected.
+Otherwise the common templates should never be modified manually, find out if you are misusing the common templates by making suc modifications.
```

Comment:
@borod108  
typo:
templates by making suc modifications. -> templates by making such modifications.

Reply:
@borod108 Is there a way to know what template was reconciled?

Reply:
typo fixed, as for which template was reconciled - not currently, I am thinking of adding this to the log, will update the runbook once there is a way.